integrations.go

```diff
@@ -325,3 +326,62 @@ func (cfi *contextifyFramesIntegration) addContextLinesToFrame(frame Frame, line
 	}
 	return frame
 }
+
+// ================================
+// Global Tags Integration
+// ================================
+
+const envTagsPrefix = "SENTRY_TAGS_"
+
+type globalTagsIntegration struct {
+	tags    map[string]string
+	envTags map[string]string
+}
+
+func (ti *globalTagsIntegration) Name() string {
+	return "GlobalTags"
+}
+
+func (ti *globalTagsIntegration) SetupOnce(client *Client) {
+	ti.tags = client.options.Tags
+	ti.envTags = loadEnvTags()
+	client.AddEventProcessor(ti.processor)
+}
+
+func (ti *globalTagsIntegration) processor(event *Event, hint *EventHint) *Event {
+	if len(ti.tags) == 0 && len(ti.envTags) == 0 {
+		return event
+	}
+
+	if event.Tags == nil {
+		event.Tags = make(map[string]string, len(ti.tags)+len(ti.envTags))
+	}
+
+	for k, v := range ti.tags {
+		if _, ok := event.Tags[k]; !ok {
+			event.Tags[k] = v
```

Comment:
Since we also update `event.Tags` in `scope.ApplyToEvent` (and potentially somewhere else?), please double check that we won't cause any "concurrent writes" panics around it.

Reply:
This should be fine and in line with other integrations from what I can tell. 

They all tend to modify the event in some way. `event.Tags` doesn't seem special in this regard. `scope.ApplyToEvent` is called when the event is prepared as part of running each event processor. (the scope seems to have it's own slice of tags which is protected for concurrent access)

Reply:
Sgtm, thanks for checking 👍